src/appMain/CMakeLists.txt

```diff
@@ -110,12 +110,6 @@ set(LIBRARIES
   ConfigProfile
   Resumption
 )
-if(REMOTE_CONTROL)
```

Comment:
is REMOTE_CONTROL still build flag that can enable/disable REMOTE_CONTROL in core?